data/presets/presets.json

```diff
@@ -8456,6 +8456,26 @@
             },
             "name": "Golf Course"
         },
+        "leisure/hackerspace": {
+            "icon": "commercial",
+            "fields": [
+                "name",
+                "address",
+                "website"
+            ],
+            "geometry": [
+                "point"
```

Comment:
Thanks @willemarcel, you can `"area"` here after `"point"`.  I know the wiki page says point only, but it's wrong.  All leisure tags that can happen in a building can be mapped as area.  (in fact the summary `leisure=` page shows area also: https://wiki.openstreetmap.org/wiki/Key:leisure )